configs/platform_cfg.yaml

```diff
@@ -26,6 +26,11 @@ platform:
   - native-type: hdb
```

Comment:
1. you are missing the binary generation
2. For java module, do you plan to add integration test?

Reply:
1. No, it's in the change (internal/platform/platform_cfg.go)
2. I tested it manually, but I didn't see any tests for other module types (that's why I didn't add tests for the new npm module types I added either). Where should I add it?